waddrmgr/db.go

```diff
@@ -1222,6 +1685,24 @@ func upgradeManager(namespace walletdb.Namespace) error {
 			return managerError(ErrDatabase, str, err)
 		}
 
+		_, err = rootBucket.CreateBucketIfNotExists(acctNameIdxBucketName)
+		if err != nil {
+			str := "failed to create account name index bucket"
+			return managerError(ErrDatabase, str, err)
+		}
+
+		_, err = rootBucket.CreateBucketIfNotExists(acctIdIdxBucketName)
+		if err != nil {
+			str := "failed to create account id index bucket"
+			return managerError(ErrDatabase, str, err)
+		}
+
+		_, err = rootBucket.CreateBucketIfNotExists(metaBucketName)
+		if err != nil {
+			str := "failed to create meta bucket"
+			return managerError(ErrDatabase, str, err)
+		}
+
```

Comment:
As mentioned elsewhere, when the meta bucket is created, it think be initialized with all counters fields at zero and the other places can then error if the field isn't there.